eng/generators.targets

```diff
@@ -8,14 +8,18 @@
   <ItemGroup>
     <EnabledGenerators Include="LibraryImportGenerator" Condition="'$(EnableLibraryImportGenerator)' == 'true'" />
     <!-- If the current project is not System.Private.CoreLib, we enable the LibraryImportGenerator source generator
-         when the project is a C# source project that:
+         when the project is a C# source or test project that:
          - doesn't target the a TFM that includes LibraryImportGenerator or
          - doesn't reference the live targeting pack (i.e. when inbox) and
            - references System.Private.CoreLib, or
            - references System.Runtime.InteropServices -->
     <EnabledGenerators Include="LibraryImportGenerator"
                        Condition="'$(EnableLibraryImportGenerator)' == '' and
-                                  '$(IsSourceProject)' == 'true' and
+                                  (
+                                    '$(IsSourceProject)' == 'true' or
+                                    '$(IsTestProject)' == 'true' or
+                                    '$(IsTestSupportProject)' == 'true'
```

Comment:
This is either true for projects that target a non .NETCoreApp TFM or ones that disable implicit framework references and then reference System.Runtime.InteropServices or CoreLib. So essentially this logic only applies to our .NET Framework TFMs in test projects.

Is that intention? Just want to better understand the change.

Reply:
Yes, this should only be true for .NET Framework or Standard TFMs in tests.